components/sup/src/manager/mod.rs

```diff
@@ -194,7 +194,7 @@ impl Default for ManagerConfig {
             auto_update: false,
             custom_state_path: None,
             update_url: "".to_string(),
-            update_channel: ChannelIdent::default(),
+            update_channel: ChannelIdent::default(), // or configured_value?
```

Comment:
`default` for sure... we don't want `Default` implementations to depend on environment variables.